src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -113,6 +113,7 @@ def load_arguments(self, _):
         c.argument('enable_private_cluster', action='store_true')
         c.argument('private_dns_zone')
         c.argument('fqdn_subdomain')
+        c.argument('enable_public_fqdn', action='store_true')
```

Comment:
Please add `is_preview=True` to the registration of arguments

Reply:
just learned that. 
It'll show `Argument '--enable-public-fqdn' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus` after registeration. Thanks.